src/main/scala/edu/ie3/simona/model/participant/SystemParticipant.scala

```diff
@@ -45,7 +44,7 @@ abstract class SystemParticipant[CD <: CalcRelevantData](
     operationInterval: OperationInterval,
     scalingFactor: Double,
     qControl: QControl,
-    sRated: squants.Power, // TODO ApparentPower type
```

Comment:
I won't do this, at least at the moment. We can leave the TODO if one might start this later. But I'm not sure if that brings value in here since squants don't differ between VA and W. But maybe I'm missing something?

Reply:
They do if we want them too at least wrt their symbol. This then would give some more insight into what we expect at which point. I opened a discussion to think about the question at some point: https://github.com/ie3-institute/simona/discussions/580